integration-tests/testconfig/default.toml

```diff
@@ -1,3 +1,8 @@
+# This is the default configuration so OCR2 tests can run without issues
+[ChainlinkImage]
+image="public.ecr.aws/chainlink/chainlink"
+version="2.9.0"
```

Comment:
are you sure you want to commit that? we usually pass that in `overrides.toml` or in base64 env var

Reply:
Yeah, usually for local development so people don't have to look around to get tests to work.